pkg/middleware/namespace_middleware.go

```diff
@@ -0,0 +1,13 @@
+package middleware
+
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+func IsNamespace(c *fiber.Ctx) error {
```

Comment:
Set to `SetNamespace` or `NormalizeNamespace`.
Functions that start with `Is...` should return `bool`